### ISSUE TITLE
just curious why no facade available?

### ISSUE THREAD

**Issue by vesper8** [spatie/laravel-html#12]
First of all wanna say big fan and big user of all of your packages. Congrats on #4 spot

The older LaravelCollective you're aiming to replace had a useful Facade.. just wondering why you opted not to include one?

I know you explain how to add a helper, but I include these dependencies as part of modules that I slap on during my app scaffolding and automating the creation of an helper is not handy. So I've gone ahead and use it this way instead in one of my slapped-on service providers's boot method:

```
    public function boot()
    {
        $this->registerProviders();
        $this->loadAliases();
        $this->scheduleCommands();

        // Don't execute anything else if running from console
        if ($this->app->runningInConsole()) {
            return;
        }

        \View::share('html', app(\Spatie\Html\Html::class));
    }
```

and then I can use it this way in any view:

```
{{ $html->form('PUT', route('signup'))->class('form-inline join')->open() }}
@include('signup::errors._summary')
<div class="form-group {{ $errors->has('email') ? 'has-error' : '' }}">
    {{ $html->email('email')->class('form-control')->placeholder('Your E-Mail') }}
</div>
{{ $html->submit('Step Inside')->class('btn btn-success') }}
{{ $html->form()->close() }}
```

Have any thoughts on this? Is this less ideal to a global helper you think?

Anyway, I would like to see a facade added for those of us who would use it and for whom creating a global helper is inconvenient.

Cheers!

**Comment by sebastiandedeyne** [comment]
One of the main reasons for writing this package is to provide easy autocompletion. Facades don't provide this, and shared variables don't either, so we prefer to use a helper function.

I don't mind shipping a facade in the package for those that prefer them though. I'll try to find some time to add one later today :)

**Comment by sebastiandedeyne** [comment]
Released in v1.2!
